mm2src/coins/eth/v2_activation.rs

```diff
@@ -418,6 +469,49 @@ pub(crate) async fn build_address_and_priv_key_policy(
                 derivation_method,
             ))
         },
+        EthPrivKeyBuildPolicy::Trezor => {
+            let path_to_coin = json::from_value(conf["derivation_path"].clone())
+                .map_to_mm(|e| EthActivationV2Error::ErrorDeserializingDerivationPath(e.to_string()))?;
+
+            let trezor_coin: Option<String> = json::from_value(conf["trezor_coin"].clone()).ok();
+            if trezor_coin.is_none() {
+                return MmError::err(EthActivationV2Error::CoinDoesntSupportTrezor);
+            }
+            let crypto_ctx = CryptoCtx::from_ctx(ctx)?;
+            let hw_ctx = crypto_ctx
+                .hw_ctx()
+                .or_mm_err(|| EthActivationV2Error::HwContextNotInitialized)?;
+            let hd_wallet_rmd160 = hw_ctx.rmd160();
+            let hd_wallet_storage = HDWalletCoinStorage::init_with_rmd160(ctx, ticker.to_string(), hd_wallet_rmd160)
+                .await
+                .mm_err(EthActivationV2Error::from)?;
+            let accounts = load_hd_accounts_from_storage(&hd_wallet_storage, &path_to_coin).await?;
+            // Todo: use fn gap_limit(&self) -> u32 { self.activation_params().gap_limit.unwrap_or(DEFAULT_GAP_LIMIT) } like UTXO
+            let gap_limit = DEFAULT_GAP_LIMIT;
```

Comment:
Should be https://github.com/KomodoPlatform/komodo-defi-framework/blob/da421b472f7f157462d954732f47ccf9ad857c2e/mm2src/coins/eth/v2_activation.rs#L453